src/styles/_stars.scss

```diff
@@ -0,0 +1,21 @@
+@import 'variables';
+
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+
+  &-star {
+    display: inline-block;
+    width: 16px;
+    height: 16px;
+    background: no-repeat url(/src/images/star.svg);
+    background-position: center;
+  }
+
+  &--active-4 {
```

Comment:
The class name '&--active-4' might not follow the BEM naming convention correctly. Consider using a more descriptive modifier name like '&--active-four' or '&--four-active'.